koku/koku/middleware.py

```diff
@@ -113,6 +114,18 @@ def __init__(self, dikt):
         super().__init__(data)
 
 
+class KokuTenantSchemaExistsMiddleware(MiddlewareMixin):
+    """A middleware to check if schema exists for Tenant."""
+
+    def process_request(self, request):
+        tenant_username = request.user.username
+        user = User.objects.get(username=tenant_username)
```

Comment:
```suggestion
        user = User.objects.select_related('customer').get(username=tenant_username)
```

Reply:
I removed the user lookup altogether because the request contains the tenant info. So there is no need to look anything up.